models/Headers.yaml

```diff
@@ -0,0 +1,28 @@
+openapi: 3.1.0
+info:
+  title: DCSA Headers
+  description: |
+    Common header components for DCSA APIs.
```

Comment:
How is stoplight handling this value in case it needs to be changed for specific APIs?

Reply:
What do you mean?